sale_order_product_recommendation/__manifest__.py

```diff
@@ -11,6 +11,7 @@
     "license": "AGPL-3",
     "application": False,
     "installable": True,
+    "maintainers": ["rafaelbn", "yajo"],
```

Comment:
@rafaelbn I think @sergio-teruel is talking about this line.

I'm proposing adoption of the module because we want to help in its maintenance. This module had no declared maintainers. See https://odoo-community.org/page/maintainer-role.

If you want, I can open this in a separate PR.

Reply:
@sergio-teruel would you agree?

Reply:
@yajo add @sergio-teruel as maintainer in the first place and let's merge this. Thank you!

Reply:
Done.